.github/workflows/rustfmt.yml

```diff
@@ -18,4 +25,10 @@ jobs:
         with:
           components: rustfmt
       - name: Rustfmt check
-        run: cargo fmt -- --check -v
+        run: cargo fmt -- -v ${{ github.event_name == 'pull_request' && '' || '--check' }}
```

Comment:
I might have gotten the syntax wrong here because the GHA log says it ran `cargo fmt -- -v --check`, whereas this should have made it `cargo fmt -- -v`. 🤔

Reply:
Maybe just add two steps with `if` conditions on the step instead?